bambi/models.py

```diff
@@ -239,6 +240,17 @@ def fit(
             in the system unless there are more than 4 CPUs, in which case it is set to 4.
         random_seed : int or list of ints
             A list is accepted if cores is greater than one.
+        sampler_backend: str
+            If ``default`` uses the standard PyMC fit() method. Is only valid if method == "mcmc".
+            Other valid sampling methods are ``numpyro`` and ``blackjax`` which both use JAX (see
+            https://www.pymc.io/blog/v4_announcement.html#new-jax-backend-for-faster-sampling)
+            and which offers both performance improvements on the CPU due to JIT compilation
+            and also GPU use (which will happen automatically if a GPU is detected - see the
+            JAX documentation https://jax.readthedocs.io/en/latest/index.html). Also, both
```

Comment:
```suggestion
            Other valid sampling methods are ``numpyro`` and ``blackjax`` which both use JAX. Also, both
```
In my opinion, this is too verbose for the docstring. But it will be very nice to have as part of an example.